src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
```

Comment:
The method 'getRandomColor' is well-named as it clearly indicates its purpose. Just make sure that the method always returns a valid color and handles the randomization appropriately.